projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -55,21 +55,21 @@
   "block.ars_nouveau.portal": "折越传送门",
   "block.ars_nouveau.arcane_core": "奥术核心",
   "block.ars_nouveau.imbuement_chamber": "灌注室",
-  "ars_nouveau.page.jar_of_light": "召唤会跟随使用者移动的光。可随时召唤和驱散。",
+  "ars_nouveau.page.jar_of_light": "召唤随使用者移动的光源，可被随时驱散。",
   "ars_nouveau.page.amulet_of_mana_boost": "中幅增加魔力上限。",
-  "ars_nouveau.page.amulet_of_mana_regen": "中幅增加魔力再生速度。",
-  "ars_nouveau.page.magebloom_crop": "一种充满魔法的花。法师之花生长时可为附近的农艺魔源通道提供额外魔源，成熟后还可产出法师之花纤维。法师之花还可以用来合成法术伤害药水。",
-  "ars_nouveau.page.belt_of_levitation": "可使玩家以中等高度浮空的腰带。爬山时很有用！掉落（或跳跃）时在空中潜行即可升起。",
-  "ars_nouveau.page.ring_of_lesser_discount": "除了为最大魔力和魔力再生提供少量加成外，节魔之戒还能降低施放法术时需要消耗的总魔力。",
-  "ars_nouveau.page.ring_of_greater_discount": "比次等节魔之戒提供稍多的魔力节省。",
-  "ars_nouveau.page.belt_of_unstable_gifts": "会偶尔给予一个随机、短期、正面的药水效果。这些效果的强度各不相同。",
-  "ars_nouveau.page.reactive": "反应是一种可以应用于§l任何§r物品的附魔，可以通过附魔装置进行附魔和升级。带有反应附魔的工具和盔甲在使用时或玩家受伤时有几率自动施法。该附魔所施放的法术取决于第一个合成阶段所抄写的法术。法术的成功施放需要使用者有足够的魔力。",
-  "ars_nouveau.page.reactive2": "将任何物品或盔甲放入附魔装置中都可附上反应附魔。放在基座上的法术羊皮纸必须抄写有一个有效的法术。更多的信息请参阅抄写台的条目。反应附魔还需要在附魔装置附近放置魔源罐。第一级附魔大约消耗魔力罐所存储魔源的三分之一。",
-  "ars_nouveau.page.reactive3": "将合成中的试剂替换为任何已经有反应附魔的物品即可升级附魔。第二级反应附魔需要放置一个有反应I附魔的物品，第三级反应附魔需要放置一个有反应II附魔的物品。附魔等级越高，过程中需要的魔源量明显提高。",
+  "ars_nouveau.page.amulet_of_mana_regen": "中幅提高魔力恢复速度。",
+  "ars_nouveau.page.magebloom_crop": "蕴含魔力的神奇花朵。法师之花生长时为农艺魔源通道提供的魔源更多，成熟后还可用于生产法师之花纤维，或制成法术伤害药水。",
+  "ars_nouveau.page.belt_of_levitation": "能使玩家以一定高度离地悬浮的魔法腰带，只需在玩家腾空时潜行即可使用。爬山时很有用！",
+  "ars_nouveau.page.ring_of_lesser_discount": "少量提升魔力上限和魔力恢复速度，并减少施放魔法时的魔力消耗。",
+  "ars_nouveau.page.ring_of_greater_discount": "比次等节魔之戒节省更多的魔力。",
+  "ars_nouveau.page.belt_of_unstable_gifts": "偶尔会随机给予玩家一个短期的正面效果，等级不定。",
+  "ars_nouveau.page.reactive": "反应是一种能被附魔装置附加在§l任何§r物品上的附魔。带有反应附魔的物品会在被使用或受击时概率释放抄录在内的法术，此次释放仍然需要正常消耗足量魔力。法术将在进行第一级附魔时被抄录进物品。",
+  "ars_nouveau.page.reactive2": "将任何物品或盔甲放入附魔装置中都可附上反应附魔。作为材料的法术羊皮纸必须抄写一个有效的法术。更多的信息请参阅抄写台的条目。附加第一级附魔大约消耗三分之一罐魔源。",
+  "ars_nouveau.page.reactive3": "将合成中的试剂替换为任何已经有反应附魔的物品即可升级附魔。第二级反应附魔需要放置一个有反应I附魔的物品，第三级反应附魔需要放置一个有反应II附魔的物品。附魔等级越高，过程中需要的魔源量也越高。",

```

```suggestion
  "ars_nouveau.page.reactive3": "将合成中的试剂替换为任何已经有反应附魔的物品即可升级附魔。第二级反应附魔需要一件有反应I附魔的物品，第三级反应附魔则需要一件有反应II附魔的物品。附魔等级越高，过程中需要的魔源量也越高。",
```
不懂就问，MC里的罗马数字是用`Ⅰ`、`Ⅱ`等表示的，还是拿英文半角字母表示的

---

不知道()
之前的翻译文本里是用的英文字母

---

确认了，大写字母